web/styles/popovers.css

```diff
@@ -1233,9 +1233,11 @@ ul {
     margin-left: auto;
     color: hsl(0deg 0% 40%) !important;
     border-radius: 4px;
+    opacity: 0.5;
```

Comment:
Why 0.5/0.9? I feel like we usually use 0.7/1 for this sort of thing.

Reply:
Sorry, missed to mention the discussion URL here - https://chat.zulip.org/#narrow/stream/101-design/topic/new.20personal.20menu/near/1671603

The change was suggested by @terpimost